packages/horizontal-slider-js/README.md

```diff
@@ -1,13 +1,13 @@
-# `@algolia/ui-components-js-horizontal-slider`
+# `@algolia/ui-components-horizontal-slider-js`
```

Comment:
much better name, alternative would have been `js-ui-components- horizontal-slider`, but this format works well